src/LibExpat.jl

```diff
@@ -1,13 +1,12 @@
 module LibExpat
 
 using Compat
-using Compat: String, unsafe_string
 
 import Base: getindex, show, parse
 
-if is_windows() 
+if Compat.Sys.iswindows() 
     const libexpat = "libexpat-1"
-elseif is_unix() 
+elseif Compat.Sys.islinux() 
```

Comment:
Now if this had a BinDeps then these would be library aliases but it does not since libexpat is shipped with julia.